packages/filters/src/Filters.tsx

```diff
@@ -211,18 +229,30 @@ const Filters = forwardRef(function Filters<T extends Record<string, unknown>>(
     Object.keys(updatedFiltersMap).forEach(key => {
       const isSticky = filtersConfig[key as FilterKeys]?.isSticky
       const defaultValue = filtersConfig[key as FilterKeys]?.defaultValue
+      const parser = filtersConfig[key as FilterKeys]?.parser
+
+      const serializedDefaultValue = defaultValue ?? parser?.serialize(defaultValue)
```

Comment:
do you think we should save serialized defaultValue in filtersConfig? parsing again will not be required.